gematria/datasets/pipelines/BUILD.bazel

```diff
@@ -25,6 +29,12 @@ gematria_py_binary(
 gematria_py_test(
     name = "compile_modules_lib_test",
     srcs = ["compile_modules_lib_test.py"],
+    # TODO(boomanaiden154): We use the python implementation here as otherwise
+    # we get a segmentation fault. Eventually this should be debugged and
+    # fixed.
```

Comment:
Please add a mention of https://github.com/google/gematria/issues/201 in here (so that we can search TODOs by issues).